broadcasting.md

```diff
@@ -188,45 +201,122 @@ window.Echo = new Echo({
 <a name="client-ably"></a>
 ### Ably
 
-[Laravel Echo](https://github.com/laravel/echo) is a JavaScript library that makes it painless to subscribe to channels and listen for events broadcast by your server-side broadcasting driver. You may install Echo via the NPM package manager. In this example, we will also install the `pusher-js` package.
-
-You may wonder why we would install the `pusher-js` JavaScript library even though we are using Ably to broadcast our events. Thankfully, Ably includes a Pusher compatibility mode which lets us use the Pusher protocol when listening for events in our client-side application:
+[Ably Laravel Echo](https://github.com/ably-forks/echo/) is a JavaScript library that makes it painless to subscribe to channels and listen for events broadcast by your server-side broadcasting driver. Ably is maintaining a fork of the official laravel-echo module which allows you to use the official [ably-js](https://github.com/ably/ably-js) SDK. In this example, we will also install the official `ably` package:
```

Comment:
I think calling it "Ably laravel echo" is a bit audacious, I think this paragraph should be structured more like:
> [Ably] maintain a fork of laravel echo which allows you to use the official [Ably JavaScript SDK]. In this example we will also install the official `ably` package:

Reply:
Okay, I will update this accordingly.

Reply:
I have updated code accordingly across different PRs.